packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -78,19 +78,27 @@ class ResizeHandle extends (Widget as any) {
       case 'next':
         return `${RESIZE_HANDLE_COLLAPSE_NEXT_PANE_CLASS} ${ICON_CLASS} ${this._getCollapseIconClass(true)}`;
       case 'icon':
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} ${this._getResizeIconClass()}`;
       default:
-        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} dx-icon-overflow`;
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} .dx-icon-overflow`;
```

Comment:
We can type incoming iconType argument. It can be 'prev', 'icon', 'next'. After this we can remove default branch.

Reply:
correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/6754eeced384460fb394c2160b6e5d628e935f80)